packages/cf-builder-pagination/example/basic/component.js

```diff
@@ -1,6 +1,5 @@
-const React = require('react');
-const { render } = require('react-dom');
-const { PaginationBuilder } = require('../../src/index');
+import React from 'react';
```

Comment:
we change from common.js to es6 modules syntax. Is it not enough to bump package version of all components?